actions/src/create-cut-pr/main.test.ts

```diff
@@ -1,8 +1,7 @@
 import type { OctokitResponse } from "@octokit/types";
+import type { MockInstance } from "vitest";
 import { describe, afterEach, beforeEach, it, vi } from "vitest";
```

Comment:
```suggestion
import { describe, afterEach, beforeEach, it, vi, type MockInstance } from "vitest";
```

May or may not be worth merging these

Reply:
This rule: https://typescript-eslint.io/rules/no-import-type-side-effects/ requires the type imports to use `import type {` so we can't merge these into one import.